en/02_Developer_Guides/14_Files/05_File_Manipulation.md

```diff
@@ -374,7 +376,7 @@ class MyFileConverter implements FileConverter
 
         // Handle conversion to PDF
         if (strtolower($toExtension) === 'pdf') {
-            return $file->manipulateExtension(
+            return $from->manipulateExtension(
```

Comment:
This is unrelated but there is no `$file` variable. It was always meant to be `$from`.